bundles/binding/org.openhab.binding.powermax/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="powermax" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>Visonic PowerMax Binding</name>
+	<description>The Visonic PowerMax binding interfaces with Visonic PowerMax and PowerMaster alarm panel series.</description>
+	<author>lolodomo</author>
```

Comment:
Because we have the real names in all other bindings as well. And it is an info shown in the UI and not just in the source code.